releng/tools/rflash

```diff
@@ -0,0 +1,45 @@
+#!python
```

Comment:
this shebang does not seem to work with bash. I suggest to use something like `#!/usr/bin/env python3`